R/vegdist.R

```diff
@@ -48,7 +48,7 @@
     if (method == 21)  # aitchison
         x <- decostand(x, "clr", ...)  # dots to pass possible pseudocount
     if (method == 22)  # robust.aitchison
-        x <- decostand(x, "rclr") # No pseudocount for rclr
+        x <- decostand(x, "rclr", nr.rm=na.rm, ...) # No pseudocount for rclr
```

Comment:
Is this a typo? Should it be `na.rm = na.rm, ...`?

Reply:
fixed